network-api/networkapi/wagtailpages/factory/publication.py

```diff
@@ -42,6 +42,41 @@ class Meta:
     file = django.FileField(filename=Faker('file_name', category="text"), file_extension="pdf")
 
 
+class PublicationPageFactory(PageFactory):
+    title = Faker('text', max_nb_chars=120)
+    subtitle = Faker('text', max_nb_chars=250)
+    secondary_subtitle = Faker('text', max_nb_chars=250)
+    publication_date = Faker('date_object')
+    hero_image = SubFactory(ImageFactory)
+    publication_file = DocumentFactory()
+
+
+# UGLY COPYPASTE FROM latest
```

Comment:
in the near/mid future, now that we have control over wagtail_factories we can eventually put out a new release and avoid this copypasta